### PR TITLE
Add guidance to proposal section

### PR DESCRIPTION
**Pull request by mhdawson**
Add additional guiance to proposal section to call out
that proposals need to explain how the project will
better the Node.js ecosystem.

### CODE REVIEW

**Comment by feross** [comment]
:+1:

**Comment by feross** [comment]
I'd also like to see a bit about why a project would want to join the foundation, i.e. what they can expect from the process and why they might want that.

File: Project-Lifecycle.md [hunk 1]
```diff
@@ -98,10 +98,26 @@ vote to abstain from representation on the TSC.
...
+For a project to be accepted as a top-level project under the Node.js
+Foundation it must help better the Node.js ecosystem.  It is important
+that proposals include a clear explanation of why this will be the
+case.  A few areas you may want to include:
+
+* how will the project help the Node.js ecosystem grow as a whole ?
+* what set of Node.js users is the project interesting/critical to ?
+* is there a direct relationship between the project and core Node.js
+  or one of the existing top-level projects?  For example is it a
+  Node.js dependency ?
```
Comment (orangemocha) [inline Project-Lifecycle.md]: I wonder if "is it a Node.js dependency" might be misinterpreted. Suggestion "does Node.js itself depend on it"

**Comment by orangemocha** [comment]
Nit: `guiance` => `guidance` in the commit message.

**Comment by orangemocha** [comment]
This looks like a useful starting point to me. LGTM, with a couple of really tiny nits.

**Comment by mikeal** [comment]
These are some good questions.

The state of the docs here need a big re-working to get in-line with what we're already doing. It's very hard to figure out the TSC is doing in practice and what the Incubator is for.

There's also a few tough questions we need to address that I'll be logging over the next few days.

This is all just to say that I think that we should merge this but shortly after re-work the README, this doc, and most of the other docs as well.

**Comment by mhdawson** [comment]
Updated to address review comments so far.

**Comment by jasnell** [comment]
LGTM

**Comment by mhdawson** [comment]
Can this be landed ?

**Comment by jasnell** [comment]
Should be safe but let's put it on the tsc-agenda for tomorrow just to be extra safe ;-) I'll land it tomorrow if there are no objections.

**Comment by Fishrock123** [comment]
TSC has decided to delay this for now while we figure some mission-y things out.

**Comment by mhdawson** [comment]
Don't think this is relevant anymore.

**Pull request closed without merging**